src/main/webapp/app/overview/header-course.component.ts

```diff
@@ -49,4 +52,13 @@ export class HeaderCourseComponent implements OnChanges {
             }
         }
     }
+
+    showGoToCourseManagementButton() {
```

Comment:
This method name sounds like it would show the button, but it actually only determines whether to show the button. Maybe you could rename it to something like `shouldShowGoToCourseManagementButton` and annotate the return value with bool?

Reply:
Done